### PR TITLE
Added default selected options in Deploy Agent page

### LINKED ISSUE #5061: [Deploy Agent Page] Set default versions and architectures
**Description**
Set default architecture when having only one on the Deploy Agent page.

We need to add auto-selected options for Version and Architecture when:
- Have only one option
- Select the default option (version, architecture) for a specifics OS.

Related issue #5042

### PR DESCRIPTION
**Pull request by Machi3mfl**
### Description
Added default selected options in Deploy Agent Page 

Closes #5061 

### Check the auto-selected version and architecture when having only one option
1. Go to Deploy agent 
2. Select any OS 
3. Check the auto-selected options when OS has only one option for architecture or version

### Check the auto-selected OS options
1. Go to Deploy agent 
2. Select any OS 
3. Check the auto-selected options for version and architecture
4. Repeat for every OS

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [x] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by Machi3mfl** [comment]
## Solution details

Created a new component called `RegisterAgentButtonGroup` with the following behavior:

- Receives and renders buttons depending on the options list with the following structure:
```json
{
  id: string,
  label: string,
  default?: boolean
}
```
- When the component receives the options list with `one option` `the button is auto-selected`
- When an option has the property `default` in `true` `the button is auto-selected`.

## Behavior evidence
Peek 2023-01-03 12-07

File: public/controllers/agent/components/register-agent.js [hunk 1]
```diff
@@ -2179,7 +2180,6 @@ apk add wazuh-agent=${this.state.wazuhVersion}-r1`,
                       this.state.selectedVersion == 'centos7' ||
```
Comment (chantal-kelm) [inline public/controllers/agent/components/register-agent.js:2180]: conditionals in disuse, they can be removed

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
Test: ✅

**Review by Tostti — APPROVED** [review]
CR ✔️
TEST ✔️

LGTM

**Review by yenienserrano — DISMISSED** [review]
LGTM

image

**Review by yenienserrano — APPROVED** [review]

**Review by chantal-kelm — DISMISSED** [review]
CR: ✅ 
Test: ✅

**Review by Tostti — DISMISSED** [review]

**Review by yenienserrano — APPROVED** [review]

**Pull request merged**